### DISCUSSION TITLE
Annotations and Sub-graph ?

### DISCUSSION THREAD

**Discussion by pravin-d** [imaNNeo/fl_chart#1471] [Q&A]
This is two part question. Both of them are related to my use case
1. Does fl_charts support point annotations ? If yes, would be able to make it clickable ?
Use case:
  Lets say I have a graph of todays temperature. I would want to mark events like heavy rain, etc on the x-axis(time axis). When I click the annotation, it should give me a popup showing some details of the event.
2. Can I have another chart below the actual chart. Use case is same as above, except I would put the annotation in the sub-graph.

A good sample would be -> Please replace the candle sticks with a simple line graph
The main graph and the volume graph are in separate panels but have a common x-axis.
There an event annotation which when click shows a popup
![Annotate-a-local-image](https://github.com/imaNNeo/fl_chart/assets/6554155/b32addb1-465d-4272-84cd-fafd72f4ae71)

**Comment by imaNNeo** [comment]
We've something similar for annotations (but not exactly what you want). It doesn't support touch callbacks yet.

You can take a look at LineChartSample8 in our [line-chart-samples](https://app.flchart.dev/#/line)
<img width="400" alt="image" src="https://github.com/imaNNeo/fl_chart/assets/7009300/9609e409-64b6-4d1b-8f52-024e55135e1b">

**Reply by imaNNeo** [reply]
About your second question, yes you can hide the titles and put the charts in a Column to show them. Don't forget to provide a height for them, because fl_chart doesn't provide a height for you. (You can use AspectRatio too)
